apps/activejobs/app/assets/javascripts/application.js

```diff
@@ -157,65 +200,68 @@ function create_datatable(options){
                                             $(nTd).addClass('details-control');
                                         }
                                     },
-                "render":           function() {
-                                        return "";
-                                    }
+                render: function () {
+                  return "";
+                },
             },
             {
                 data:               "pbsid",
                 className:          "small",
                 "autoWidth":        true,
-                "render":           function(data) {
-                    return "<span title="+data+">"+data+"</span>";
-                }
+                render: function (data) {
+                  return `<span title="${data}">${data}</span>`;
+                },
             },
             {
                 data:               "jobname",
                 className:          "small",
                 width:              '25%',
-                "render":           function(data) {
-                    return "<span title="+data+">"+data+"</span>";
-                }
+                render: function (data) {
+                  return `<span title="${data}">${data}</span>`;
+                },
             },
             {
                 data:               "username",
                 className:          "small",
                 "autoWidth":        true,
-                "render":           function(data) {
-                    return "<span title="+data+">"+data+"</span>";
-                }
+                render: function (data) {
+                  return `<span title="${data}">${data}</span>`;
+                },
             },
             {
                 data:               "account",
                 className:          "small",
                 "autoWidth":        true,
-                "render":           function(data) {
-                    return "<span title="+data+">"+data+"</span>";
-                }
+                render: function (data) {
+                  return `<span title="${data}">${data}</span>`;
+                },
             },
             {
                 data:               "walltime_used",
                 className:          "small text-right",
                 "autoWidth":        true,
-                "render":           function(data) {
-                    return "<span title="+human_time(data)+">"+human_time(data)+"</span>";
-                }
+                render: function (data) {
+                  return `
+                    <span title="${human_time(data)}">
+                      ${human_time(data)}
+                    </span>
+                  `;
+                },
             },
             {
                 data:               "queue",
                 className:          "small",
                 "autoWidth":        true,
                 "render":           function(data) {
```

Comment:
Above you changed `"render":` to `render:` but here you keep it the same. Should we just update everything to be consistent?